### PR TITLE
feat: Add Sentry integration

### PR DESCRIPTION
**Pull request by taion**

### CODE REVIEW

File: tqp/raven.py [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+# -----------------------------------------------------------------------------
+
+
+def install(client):
+    base_handle_error = QueuePollerBase.handle_error
+
+    def _hook_handle_error(self, exception, msg, payload):
+        base_handle_error(self, exception, msg, payload)
+
+        client.captureException(extra={"payload": payload})
```
Comment (itajaja) [inline tqp/raven.py]: `"msg": msg` as well?

File: tqp/raven.py [hunk 2]
```diff
@@ -9,6 +9,6 @@ def install(client):
     def _hook_handle_error(self, exception, msg, payload):
         base_handle_error(self, exception, msg, payload)
 
-        client.captureException(extra={"msg": msg})
```
Comment (taion) [inline tqp/raven.py]: @itajaja the problem is that this is actually some opaque object from moto, so nothing useful will go to sentry

Comment (itajaja) [reply tqp/raven.py]: oh ok. as long as payload contains the queue name?

Comment (taion) [reply tqp/raven.py]: well, it has `topic = body['TopicArn'].split(':')[-1]`

**Pull request merged**